Updated README

Abhinav-ark opened the pull request:
Adding Documentation to README (In Progress...)

Closes #177

Abhinav-ark committed:
Update README.md

Abhinav-ark committed:
Added videos in README.md for documentation purpose

Thanus-Kumaar committed:
Updated README.md to include

Thanus-Kumaar committed:
Modified the README.md to change eventide video and added new videos too.

Thanus-Kumaar committed:
Updated README.md to include the necessary details..

Thanus-Kumaar commented:
Made the necessary changes to the readme.md, do check @Ashrockzzz2003 @Abhinav-ark

Thanus-Kumaar committed:
Fixed buggy video links

Ashrockzzz2003 reviewed (changes requested):
Awesome work @Thanus-Kumaar 
Let's add links to other repos here. 

Backend, Mobile app, admin app, qr scan ... etc

Abhinav-ark commented on README.md:
```diff
@@ -1,38 +1,131 @@
+
+---
+
+## 💳 View Transactions
+
+Manage and view your transactions effortlessly.  
```
The Video is to `View Registration and Submit Feedback`. Transactions Will be In Profile Page.

Abhinav-ark reviewed (commented):
@Thanus-Kumaar  `Profile Page` and `View Team Event Registation Details` Are Missing.

Thanus-Kumaar committed:
Added links to view other repos

Thanus-Kumaar committed:
Added profile page and registered events.

I could not add the registration details for team events as the registrations are closed and the button is disabled.

Ashrockzzz2003 committed:
Add more anokha repo links.

Ashrockzzz2003 reviewed (approved):
LGTM

Abhinav-ark commented on README.md:
```diff
@@ -1,38 +1,156 @@
+
+---
+
+## 💳 View Transactions
```
This is not just transactions. Change Emoji also.

Abhinav-ark commented on README.md:
```diff
@@ -1,38 +1,156 @@
+
+---
+
+## Profile Page
```
Why does this h2 not have an emoji? Use 👤.

Abhinav-ark commented on README.md:
```diff
@@ -1,38 +1,156 @@
+
+
+
+https://github.com/user-attachments/assets/ed589656-b210-4b45-a697-c43e7c216e2d
```
View Transactions functionality not showcased in profile page.

Abhinav-ark reviewed (commented):
Look into following changes.
